examples/utils/bundle-size-tests/webpack.config.cjs

```diff
@@ -118,8 +118,9 @@ module.exports = {
 			 * IMPORTANT: Do not add any new exceptions to this list without first doing a deep investigation on why a PR adds a new duplication, this hides a bundle size issue
 			 */
 			exclude: (instance) =>
-				// object-is depends on es-abstract 1.18.0-next, which does not satisfy the semver of other packages. We should be able to remove this when es-abstract moves to 1.18.0
-				instance.name === "es-abstract",
```

Comment:
this package no longer needs this exclusion, so I just replaced it with the new one.